model/src/main/scala/aqua/model/ServiceModel.scala

```diff
@@ -3,4 +3,9 @@ package aqua.model
 import aqua.types.ArrowType
 import cats.data.NonEmptyMap
 
-case class ServiceModel(name: String, arrows: NonEmptyMap[String, ArrowType]) extends Model
+// no service id
+case class ServiceModel(
+  name: String,
+  arrows: NonEmptyMap[String, ArrowType],
+  serviceId: Option[ValueModel]
```

Comment:
This limits `serviceId` to be defined only in the service definition. But semantics is just that `serviceId` is defined in the root scope. `SetServiceIdModel` should be a distinct one.